runtime/android/core/src/org/xwalk/core/XWalkLibraryListener.java

```diff
@@ -8,6 +8,7 @@
     public enum LibraryStatus {
         MATCHED,
         NOT_FOUND,
+        VERIFY_ERROR,
```

Comment:
`SIGNATURE_CHECK_ERROR` sounds more descriptive. Using a verb like "verify" in an enum is not usual and does not sound good.